source/common/upstream/cluster_manager_impl.h

```diff
@@ -357,6 +358,7 @@ class ClusterManagerImpl : public ClusterManager, Logger::Loggable<Logger::Id::u
   ClusterManagerInitHelper init_helper_;
   Config::GrpcMuxPtr ads_mux_;
   LoadStatsReporterPtr load_stats_reporter_;
+  HDSReporterPtr hds_reporter_;
```

Comment:
Nit: to match Envoy style, I'd name this `HdsReporterPtr`. I think the name `HdsChecker` is probably a little bit more descriptive, or maybe even `HdsDelegate` (bikeshed ahoy!).